packages/webapp/src/components/cards/cardContainer.tsx

```diff
@@ -1,12 +1,15 @@
 import React from "react"
-import { CardPlacement } from "src/store/types"
+
 import { useSortable } from "@dnd-kit/sortable"
-import DraggedCard from "./draggedCard"
-import BoardCard from "./boardCard"
-import HandCard from "./handCard"
 import { CSS } from "@dnd-kit/utilities"
+
+import { CardPlacement } from "src/store/types"
 import { convertStringToSafeNumber } from "src/utils/js-utils"
 
+import BoardCard from "./boardCard"
+import DraggedCard from "./draggedCard"
+import HandCard from "./handCard"
```

Comment:
These should use the `src/` format. I know it's not formatting related, but could you fix it?

I think we could use [this type of rule](https://stackoverflow.com/questions/71118709/eslint-only-allow-relative-import-paths-not-absolute) to prevent them, but I wonder if we'd need to be able to make exception to import files form other packages (e.g. deployment addresses) — I think we could get around that by copying files however. Could you look into it?

Reply:
added this rule, running `make check` and `make format` doesn't pop up any errors.